jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -350,6 +373,10 @@ def user_api(self):
                                            'export_bg', 'export_bg_img', 'export_bg_sub',
                                            'ext_dataset', 'ext_trace', 'ext_type',
                                            'ext_width', 'ext_add_results',
+                                           'horne_ext_profile_selected',
```

Comment:
Let's expose the SelectPluginComponent instead of just the traitlet

```suggestion
                                           'horne_ext_profile',
```